src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.html

```diff
@@ -109,11 +117,17 @@
         <fa-icon [icon]="!isCoherent(node) ? 'exclamation-circle' : 'exclamation-triangle'" 
           [title]="!isCoherent(node) ? 'A newer build of this same repository is in the tree' : (hasIncoherentDependencies(node) ? 'This build relies on dependencies that are incoherent' : '')" 
           [style.visibility]="(!isCoherent(node) || hasIncoherentDependencies(node)) ? 'visible' : 'hidden'"></fa-icon>
+        <fa-icon [icon]="hasCycle(node) ? 'redo' : 'exclamation-triangle'" 
```

Comment:
Alignment (in your image) appears to be offset from the header.

Reply:
Off to figure out how to change the alignment.

Reply:
How's this:

![image](https://user-images.githubusercontent.com/9666644/66160184-729e7b00-e5de-11e9-8e28-649b9c5fc41e.png)

Reply:
Seems fine to me.